src/System.Private.CoreLib/shared/System/Span.Fast.cs

```diff
@@ -159,6 +159,24 @@ internal Span(ref T ptr, int length)
 #endif
         }
 
+        public T this[Index index]
```

Comment:
`ref T` ?

Reply:
I following up with @agocke if this will be ok from the compiler side and then change it as you suggested.

Reply:
I confirmed that would be ok, so I'll go ahead and apply your suggestion.

Reply:
Yup, this is good from the language side.